homeassistant/components/wake_on_lan/switch.py

```diff
@@ -86,6 +86,10 @@ def __init__(
             Script(hass, off_action, name, domain) if off_action else None
         )
         self._state = False
+        self._is_dummy = False
```

Comment:
Call this `_assumed_state`. And also return it from the `assumed_state` entity property. See https://developers.home-assistant.io/docs/core/entity#generic-properties

Reply:
self._assumed_state = host is None

Reply:
When assumed state, polling should be disabled.

Reply:
Also pass `False` as 2nd arg to `async_add_entities` so it won't poll on startup.